src/runtime/description.ts

```diff
@@ -158,6 +162,20 @@ export class Description {
     if (!store) {
       return undefined;
     }
+    if (Flags.useNewStorageStack) {
+      const proxy = new StorageProxy('id', await store.activate(), store.type);
+      const handle = unifiedHandleFor({proxy, idGenerator: null, particleId: 'dummy'});
```

Comment:
we should make particleId optional as well

Reply:
it should be a key, not a particleId; but it can't be optional because this is the unique write key that ends up in the version map, I think.

Reply:
sounds good, we can rename it to something like crtdKey. I was thinking if none is passed in, we could generate it when creating the handle using the idgenerator.